src/main/java/com/checkmarx/flow/service/BitBucketService.java

```diff
@@ -37,6 +42,10 @@
     private static final String BUILD_SUCCESSFUL = "SUCCESSFUL";
     private static final String BUILD_FAILED = "FAILED";
 
+    private static final String FILE_CONTENT = "/src/{hash}/{config}";
+    private static final String HTTP_BODY_IS_NULL = "HTTP Body is null for src api ";
```

Comment:
It's better to log a more informative message, e.g. "Unable to download config-as-code file. Response body is null".

Reply:
Done.